src/action/base/create_user.rs

```diff
@@ -239,73 +239,104 @@ impl Action for CreateUser {
-            .args([".", "-create", &format!("/Users/{name}")])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[".", "-create", &format!("/Users/{name}")])
+        .await?;
+
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "UniqueID",
+        &format!("{uid}"),
+    ])
     .await?;
-    execute_command(
-        Command::new("/usr/bin/dscl")
-            .process_group(0)
-            .args([
-                ".",
-                "-create",
-                &format!("/Users/{name}"),
-                "UniqueID",
-                &format!("{uid}"),
-            ])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "PrimaryGroupID",
+        &format!("{gid}"),
+    ])
     .await?;
-    execute_command(
-        Command::new("/usr/bin/dscl")
-            .process_group(0)
-            .args([
-                ".",
-                "-create",
-                &format!("/Users/{name}"),
-                "PrimaryGroupID",
-                &format!("{gid}"),
-            ])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "NFSHomeDirectory",
+        "/var/empty",
+    ])
     .await?;
-    execute_command(
-        Command::new("/usr/bin/dscl")
-            .process_group(0)
-            .args([
-                ".",
-                "-create",
-                &format!("/Users/{name}"),
-                "NFSHomeDirectory",
-                "/var/empty",
-            ])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "UserShell",
+        "/sbin/nologin",
+    ])
     .await?;
-    execute_command(
-        Command::new("/usr/bin/dscl")
-            .process_group(0)
-            .args([
-                ".",
-                "-create",
-                &format!("/Users/{name}"),
-                "UserShell",
-                "/sbin/nologin",
-            ])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "IsHidden",
+        "1",
+    ])
     .await?;
-    execute_command(
-        Command::new("/usr/bin/dscl")
-            .process_group(0)
-            .args([".", "-create", &format!("/Users/{name}"), "IsHidden", "1"])
-            .stdin(std::process::Stdio::null()),
-    )
+    execute_command_retry_on_enotyetimplemented(&[
+        ".",
+        "-create",
+        &format!("/Users/{name}"),
+        "RealName",
+        name,
+    ])
```

Comment:
this one looks new in addition to the existing ones? Was it just missing before? Just checking...